packages/core/src/TableSection/TableSection.tsx

```diff
@@ -0,0 +1,53 @@
+import { forwardRef } from "react";
+
+import { HvBaseProps } from "../types/generic";
+import { ExtractNames } from "../utils/classes";
+import { useDefaultProps } from "../hooks/useDefaultProps";
+import { useUniqueId } from "../hooks/useUniqueId";
+import { staticClasses, useClasses } from "./TableSection.styles";
+import { HvSection, HvSectionProps } from "../Section";
+
+export { staticClasses as tableSectionClasses };
+
+export type HvTableSectionClasses = ExtractNames<typeof useClasses>;
+
+export interface HvTableSectionProps extends HvBaseProps<HTMLDivElement> {
+  sectionProps?: HvSectionProps;
+  /** A Jss Object used to override or extend the styles applied to the empty state component. */
+  classes?: HvTableSectionClasses;
+}
+
+/**
+ *
+ */
+export const HvTableSection = forwardRef<HTMLDivElement, HvTableSectionProps>(
+  (props, ref) => {
+    const {
+      id,
+      sectionProps,
+      classes: classesProp,
+      className,
+      children,
+    } = useDefaultProps("HvTableSection", props);
+
+    const { classes } = useClasses(classesProp);
+    const elementId = useUniqueId(id, "hvTableSection");
+
+    return (
+      <HvSection
+        ref={ref}
+        id={elementId}
+        className={className}
+        classes={{
```

Comment:
If I'm not mistaken, all our classes will be overridden if the user provides a `classes` object inside `sectionProps`. To avoid this we could simply do:

```tsx
export type HvTableSectionProps = HvSectionProps
```

and then use `{...others}` instead of `{...sectionProps}`.


I also noticed that sometimes the header can have [a shadow underneath](https://designsystem.hitachivantara.com/6c705d900/p/29efb2-table/b/80c89a/i/269215643). We could have a prop for this:

```tsx
export interface HvTableSectionProps extends HvSectionProps {
   raisedHeader?: boolean; // name example, couldn't find a better one at the moment
}
```